.github/workflows/go.yml

```diff
@@ -300,18 +300,22 @@ jobs:
         with:
           go-version: ${{ matrix.go }}
           cache: true
-          cache-dependency-path: go/go.sum
+          cache-dependency-path: go/go.sum      
       - name: Brew Install Arrow and pkg-config
         shell: bash
         run: brew install apache-arrow pkg-config
       - name: Install staticcheck
         run: go install honnef.co/go/tools/cmd/staticcheck@${{ matrix.staticcheck }}
       - name: Build
         shell: bash
-        run: ci/scripts/go_build.sh $(pwd)
+        run: |
+          export PKG_CONFIG_PATH=/usr/local/Cellar/openssl@3/3.0.8/lib/pkgconfig:$PKG_CONFIG_PATH
```

Comment:
Hmm, hardcoding the path like this seems brittle. Are we missing a `brew link` or something similar?

Reply:
Ah, OpenSSL is special in that regard for Homebrew...

Reply:
From what I found, it seems like it's explicitly an issue with the openssl brew package not placing `openssl.pc` in the same spot as other pkg-config paths. :( If there's a `brew link` or something that we can do, I haven't been able to find it.

Reply:
IIRC we have something similar hardcoded in a bunch of places because there is no better solution?

Reply:
Yeah, that makes sense. Oh well...